### PR TITLE
[ Selenium ] Update usecases in "PhpAssistantFeaturesTest" for catching known bugs #10117 #10698

### PR DESCRIPTION
**Pull request by Ohrimenko1988**
### What does this PR do?
Update usecases in "PhpAssistantFeaturesTest" for catching known bugs #10117 #10698

**Note!**
Related to issue https://github.com/eclipse/che/pull/11261 , should be merged after this issue.

### What issues does this PR fix or reference?
Issue: https://github.com/eclipse/che/issues/11281

### CODE REVIEW

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/languageserver/csharp/CSharpFileAdvancedOperationsTest.java [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    projectExplorer.waitItem(PROJECT_NAME + "/bin");
+  }
+
+  @Test(alwaysRun = true)
+  public void checkHoveringFeature() {
+    String expectedTextInHoverPopUp =
+        "System.Console\nRepresents the standard input, output, and error streams for console applications. This class cannot be inherited.";
+    editor.moveCursorToText("Console");
+    try {
+
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/languageserver/csharp/CSharpFileAdvancedOperationsTest.java]: redundant empty line

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/CodenvyEditor.java [hunk 2]
```diff
@@ -450,6 +450,25 @@ public String getTextFromHoverPopup() {
...
   }
 
+  /** wait full matching of text in hover popup */
+  public void waitTextInHoverPopUpEqualsTo(String expectedText) {
+    // waits popup body visibility
+    try {
+      seleniumWebDriverHelper.waitVisibility(hoverPopup);
+    } catch (TimeoutException ex) {
+      // remove try-catch block after issue has been resolved
+      fail("Known issue https://github.com/eclipse/che/issues/10674", ex);
```
[the remark this answers is not in the document]
Comment (Ohrimenko1988) [reply selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/CodenvyEditor.java]: This point comes from issue #11261 and related to changes in it.

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/CodenvyEditor.java [hunk 3]
```diff
@@ -450,6 +450,25 @@ public String getTextFromHoverPopup() {
...
+    }
+
+    // waits until text in popup is equals to specified
+    try {
+      seleniumWebDriverHelper.waitTextEqualsTo(hoverPopup, expectedText);
+    } catch (TimeoutException ex) {
+      // remove try-catch block after issue has been resolved
+      fail("Known issue https://github.com/eclipse/che/issues/10117", ex);
```
[the remark this answers is not in the document]
Comment (Ohrimenko1988) [reply selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/CodenvyEditor.java]: This point comes from issue #11261 and related to changes in it.

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/languageserver/csharp/CSharpFileAdvancedOperationsTest.java [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+        workspace.getId(), Paths.get(resource.toURI()), PROJECT_NAME, DOT_NET);
+    ide.open(workspace);
+    projectExplorer.openItemByPath(PROJECT_NAME);
+    projectExplorer.openItemByPath(PATH_TO_DOT_NET_FILE);
+    consoles.waitExpectedTextIntoConsole(FINISH_LANGUAGE_SERVER_INITIALIZATION_MESSAGE);
+    projectExplorer.waitItem(PROJECT_NAME + "/obj");
+    projectExplorer.waitItem(PROJECT_NAME + "/bin");
+  }
+
+  @Test(alwaysRun = true)
```
[the remark this answers is not in the document]
Comment (Ohrimenko1988) [reply selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/languageserver/csharp/CSharpFileAdvancedOperationsTest.java]: This point comes from issue #11261 and related to changes in it.

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/languageserver/php/PhpAssistantFeaturesTest.java [hunk 5]
```diff
@@ -127,7 +127,12 @@ public void findReferenceNodeShouldBeDisplayed() {
 
     editor.goToCursorPositionVisible(15, 8);
     menu.runCommand(ASSISTANT, FIND_REFERENCES);
-    findReferencesConsoleTab.waitReferenceWithText(EXPECTED_REFERENCE_TEXT);
+    try {
+      findReferencesConsoleTab.waitReferenceWithText(EXPECTED_REFERENCE_TEXT);
+    } catch (TimeoutException ex) {
+      // remove try-catch block after issue has been resolved
+      fail("Known permanent failure https://github.com/eclipse/che/issues/10698", ex);
```
[the remark this answers is not in the document]
Comment (Ohrimenko1988) [reply selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/languageserver/php/PhpAssistantFeaturesTest.java:134]: This point comes from issue #11261 and related to changes in it.

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/languageserver/csharp/CSharpFileAdvancedOperationsTest.java (hunk 1 above)
Comment (Ohrimenko1988) [reply selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/languageserver/csharp/CSharpFileAdvancedOperationsTest.java]: This point comes from issue #11261 and related to changes in it.

**Review by SkorikSergey — APPROVED** [review]

**Review by dmytro-ndp — APPROVED** [review]

**Comment by dmytro-ndp** [comment]
ci-build

**Pull request merged**